addons/l10n_ke/views/account_tax_views.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_ke_inherit_view_tax_tree" model="ir.ui.view">
+        <field name="name">l10n.ke.account.tax.tree</field>
+        <field name="model">account.tax</field>
+        <field name="inherit_id" ref="account.view_tax_tree" />
+        <field name="arch" type="xml">
+            <field name="description" position="after">
+                <field name="l10n_ke_item_code_id" optional="hide"/>
+            </field>
+        </field>
+    </record>
+
+    <record id="l10n_ke_inherit_view_account_tax_search" model="ir.ui.view">
+        <field name="name">l10n.ke.inherit.account.tax.search</field>
+        <field name="model">account.tax</field>
+        <field name="inherit_id" ref="account.view_account_tax_search" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='company_id']" position="after">
+                <field name="l10n_ke_item_code_id"/>
```

Comment:
do not do this in the search.  It will just bother other localizations in multti-company.